main/dnsmasq/APKBUILD

```diff
@@ -1,50 +1,54 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dnsmasq
 pkgver=2.76
-pkgrel=0
-pkgdesc="Small forwarding DNS server"
+pkgrel=1
+pkgdesc="A lightweight DNS, DHCP, RA, TFTP and PXE server"
 url="http://www.thekelleys.org.uk/dnsmasq/"
 arch="all"
 license="GPL2"
-depends=
+depends=""
 makedepends="linux-headers"
 install="$pkgname.pre-install $pkgname.pre-upgrade"
 subpackages="$pkgname-doc"
 source="http://www.thekelleys.org.uk/dnsmasq/$pkgname-$pkgver.tar.gz
 	$pkgname.initd
 	$pkgname.confd
 	"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	default_prepare || return 1
+
+	cd "$builddir"
+	sed -Ei 's|#(conf-dir=/etc/dnsmasq.d/,\*\.conf)|\1|' dnsmasq.conf.example
```

Comment:
Not needed here, this sed is not mandatory. I haven't removed any existing one, this line is new. We don't have any rule saying "put `|| return 1` after every single line of apkbuild".

I should finally fix abuild to run with `set -e` to end this nonsense cargo cult...